### PR TITLE
adding compat data for Comment

### PR DESCRIPTION
**Pull request by katzrkool**

### CODE REVIEW

File: api/Comment.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "api": {
+    "Comment": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Comment",
+        "support": {
+          "chrome": {
+            "version_added": "1"
```
Comment (jpmedley) [inline api/Comment.json]: `chrome_android` is missing. It should be `true`.

File: api/Comment.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            "version_added": true
+          }
+        }
+      },
+      "Constructor": {
+        "__compat": {
+          "description": "<code>Comment()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Comment.json]: `true`

There should also be`"chrome_android": true`

File: api/Comment.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "24"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Comment.json]: `true`

File: api/Comment.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "24"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Comment.json]: ditto

**Comment by katzrkool** [comment]
fixed, thanks

**Pull request merged**